packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -48,36 +46,46 @@ $domain-picker__suggestion-item-height: 24px;
 	}
 }
 
-.domain-picker__free-text {
-	color: var( --studio-green-40 );
-}
-
 .domain-picker__suggestion-item-group {
 	flex-grow: 1;
 }
 
-.domain-picker__suggestion-none {
+.domain-picker__suggestion-item {
 	@include domain-picker-package-medium-text;
-	line-height: 20px; // matching design
 	display: flex;
 	justify-content: space-between;
-	align-items: flex-start;
+	align-items: center;
 	width: 100%;
-	border-radius: 0;
-	padding: 10px 0;
-	margin-bottom: 7px;
+	min-height: 36px + ( 10px * 2 ) + ( 1px * 2 ); // button height + padding height + border height
+	border: 1px solid var( --studio-gray-5 );
+	padding: 10px 14px;
+	position: relative;
+	z-index: 1;
+	text-align: left;
 	cursor: pointer;
-}
 
-.domain-picker__suggestion-item {
-	// Increasing specificity because @wordpress/components stylesheets are loaded after gutenboarding stylesheets.
-	// See https://github.com/Automattic/wp-calypso/pull/38554/commits/e1f9673bcfd9eaa6469a0cfecda9b915a520961a
-	// See https://github.com/WordPress/gutenberg/pull/19535
-	@extend .domain-picker__suggestion-none;
+	&.placeholder {
+		cursor: default;
+	}
 
-	border: 1px solid var( --studio-gray-5 );
-	padding: 14px;
-	margin-bottom: 0;
+	&:hover,
+	&:focus {
+		&:not( .placeholder ) {
+			background: var( --studio-blue-0 );
+			border-color: var( --studio-blue-40 );
+			z-index: 2;
```

Comment:
Let's move `z-index` to variable so that it'll be easier to manage these at scale.

See https://github.com/Automattic/wp-calypso/blob/f1ebe9cb688a722251a77ebb4c2ab4705fa92d8f/client/landing/gutenboarding/_z-index.scss

Reply:
The problem is that this is a package and it will be hard to use these mixins.

Reply:
Ahh, fair enough! Ok as is. :-)

Reply:
The z-stack here is localized it's just a competition between the z-stack of sibling elements so alternating between `z-index: 1` and `z-index: 2` is easier to reason with.